### ISSUE TITLE
speaker classification loss converge

### ISSUE THREAD

**Issue by Kristopher-Chen** [KinglittleQ/pitch-net#4]
Hi, how does the speaker classification loss look like on your side? It diverges in the middle of the training process in my experiment. Any idea about how to balance the speaker classification loss for the classification network(positive) and the main network(negative)?

See the training loss curve for speaker classification below. (My network follows the ideas of speaker classification and pitch regression modules, but not the exact one as your task.)

![image](https://user-images.githubusercontent.com/38187438/147309541-ab6278b9-ba5a-45cb-8c30-06c1de9f01ce.png)

Thank you!

**Comment by KinglittleQ** [comment]
I think the loss is reasonable. I've observed similar loss curves.
